`make check` doesn't work on vanilla Ubuntu 22.04

⇒ Should we switch from `python` and `pytest` to `python3` and `pytest-3`?

-----

1. Command `python` is not available by default on a vanilla installation. Package [`python3-minimal`](https://packages.ubuntu.com/jammy/python3-minimall) provides `python3`, and package [`python-is-python3`](https://packages.ubuntu.com/jammy/python-is-python3), which is not installed by default, provides `python`:
  ```
  $ make check-distutils
  python setup.py check --restructuredtext --strict
  make: python: No such file or directory
  make: *** [Makefile:51: check-distutils] Error 127
  $ 
  ```
  We call `python` here:
  https://github.com/codespell-project/codespell/blob/458de804cde608110fc1ae91e4c39d92380eb0e6/Makefile#L50-L51

-----

2. Command `pytest` is not available, instead package [`python3-pytest`](https://packages.ubuntu.com/jammy/python3-pytest) provides `pytest-3`:
  ```
  $ make pytest
  pytest codespell_lib
  make: pytest: No such file or directory
  make: *** [Makefile:57: pytest] Error 127
  $ 
  ```
  We call `pytest` here:
  https://github.com/codespell-project/codespell/blob/458de804cde608110fc1ae91e4c39d92380eb0e6/Makefile#L56-L57

--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,12 @@ flake8:
 	flake8
 
 pytest:
-	pytest codespell_lib
+	@if command -v pytest > /dev/null; then \
+		pytest codespell_lib; \
+	else \
+		echo "Test dependencies not present, install using 'pip install -e \".[dev]\"'"; \
+		exit 1; \
+	fi
 
 clean:
 	rm -rf codespell.1

